test/jobs/app_uninstalled_job_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
+
+class AppUninstalledJobTest < ActiveJob::TestCase
+  def setup
+    @shop = shops(:regular_shop)
+  end
+
+  def job
+    @job ||= ::AppUninstalledJob.new
+  end
+
+  test "AppUninstalledJob marks the shop as uninstalled from the app" do
+    job.perform(shop_domain: @shop.shopify_domain)
+
+    assert_raises ActiveRecord::RecordNotFound do
+      @shop.reload
+    end
+  end
```

Comment:
Can we add another test here to make sure we don't delete any other shop than the one specified?

Reply:
Just updated the tests